book/doctrine.rst

```diff
@@ -951,8 +951,6 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
                 products:
                     targetEntity: Product
                     mappedBy: category
-            # don't forget to init the collection in the __construct() method
```

Comment:
I think this comment is kind of usefull (and not everyone uses `doctrine:generate:entities`, even more it's better to never use this command).